python/paddle/fluid/layers/io.py

```diff
@@ -445,6 +446,86 @@ def random_data_generator(low, high, shapes, lod_levels, for_parallel=True):
     return monkey_patch_reader_methods(main_prog_var)
 
 
+def py_reader(capacity, shapes, lod_levels, dtypes):
```

Comment:
set defalut value for `lod_levels` , some users can ignore this argument.

Reply:
LGTM